src/main/java/edu/cornell/kfs/fp/businessobject/options/PaymentMethodValuesFinderNonInterdepartmental.java

```diff
@@ -1,10 +0,0 @@
-package edu.cornell.kfs.fp.businessobject.options;
-
-import edu.cornell.kfs.fp.CuFPConstants;
-
-
-public class PaymentMethodValuesFinderNonInterdepartmental extends PaymentMethodValuesFinder {
-    static {
-        filterCriteria.put(CuFPConstants.INTERDEPARTMENTAL_PAYMENT, CuFPConstants.NO);
```

Comment:
This subclass was modifying the static filterCriteria Map on the superclass once loaded by the classloader. If there are documents which use PaymentMethodValuesFinder specifically, then we need to make sure that this class removal does not mess up their drop-down values.

Reply:
I'm okay with this class removal moving forward, but the functional users will need to verify that the drop-down values are still correct for the places that depend on the base PaymentMethodValuesFinder (such as Payment Request and Credit Memo documents).